.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ commands:
             git config user.email "circleci@example.com"
             git config user.name "CircleCI"
       - build-tools/merge-with-parent:
-          parent: main
+          parent: rgc/feature-branch-maven-java
```

Comment:
should this be reverted to `main` when we merge it?

Reply:
absolutely, create you spotted it.

Reply:
```suggestion
          parent: main
```